erica_app/tests/request_processing/test_erica_input.py

```diff
@@ -217,214 +217,488 @@ def test_if_widowed_lived_separated_and_corresponding_date_then_raise_no_excepti
+        try:
+            FormDataEst.parse_obj(standard_est_data)
+        except ValidationError as e:
+            pytest.fail("parse_obj failed with unexpected ValidationError " + str(e))
 
-    def test_if_person_b_has_merkzeichen_g_then_set_correct_field(self, standard_est_data):
+    def test_if_person_b_requests_fahrkostenpauschale_and_has_merkzeichen_g_set_then_raise_no_validation_error(self, standard_est_data):
+        standard_est_data['person_b_requests_fahrkostenpauschale'] = True
+        standard_est_data['person_b_disability_degree'] = None
+        standard_est_data['person_b_has_pflegegrad'] = None
+        standard_est_data['person_b_has_merkzeichen_bl'] = None
+        standard_est_data['person_b_has_merkzeichen_tbl'] = None
+        standard_est_data['person_b_has_merkzeichen_h'] = None
         standard_est_data['person_b_has_merkzeichen_g'] = True
-        resulting_input_data = FormDataEst.parse_obj(standard_est_data)
-        assert resulting_input_data.person_b_has_merkzeichen_g_ag is True
+        standard_est_data['person_b_has_merkzeichen_ag'] = None
 
-    def test_if_person_b_has_merkzeichen_g_then_do_not_set_other_merkzeichen_field(self, standard_est_data):
-        standard_est_data['person_b_has_merkzeichen_g'] = True
-        resulting_input_data = FormDataEst.parse_obj(standard_est_data)
-        assert resulting_input_data.person_b_has_merkzeichen_bl_tbl_h_pflegegrad is None
+        try:
+            FormDataEst.parse_obj(standard_est_data)
+        except ValidationError as e:
+            pytest.fail("parse_obj failed with unexpected ValidationError " + str(e))
 
-    def test_if_person_b_has_merkzeichen_ag_then_set_correct_field(self, standard_est_data):
+    def test_if_person_b_requests_fahrkostenpauschale_and_has_merkzeichen_ag_set_then_raise_no_validation_error(self, standard_est_data):
+        standard_est_data['person_b_requests_fahrkostenpauschale'] = True
+        standard_est_data['person_b_disability_degree'] = None
+        standard_est_data['person_b_has_pflegegrad'] = None
+        standard_est_data['person_b_has_merkzeichen_bl'] = None
+        standard_est_data['person_b_has_merkzeichen_tbl'] = None
+        standard_est_data['person_b_has_merkzeichen_h'] = None
+        standard_est_data['person_b_has_merkzeichen_g'] = None
         standard_est_data['person_b_has_merkzeichen_ag'] = True
-        resulting_input_data = FormDataEst.parse_obj(standard_est_data)
-        assert resulting_input_data.person_b_has_merkzeichen_g_ag is True
 
-    def test_if_person_b_has_merkzeichen_ag_then_do_not_set_other_merkzeichen_field(self, standard_est_data):
-        standard_est_data['person_b_has_merkzeichen_ag'] = True
-        resulting_input_data = FormDataEst.parse_obj(standard_est_data)
-        assert resulting_input_data.person_b_has_merkzeichen_bl_tbl_h_pflegegrad is None
+        try:
+            FormDataEst.parse_obj(standard_est_data)
+        except ValidationError as e:
```

Comment:
In these test cases, you're still waiting for a ValidationError instead of a ValueError btw. And maybe we can even get rid of the try except block in the future. There was a time I liked it, but I'm growing less fond of it because you don't see the actual error in case the test fails but rather a self-constructed error message.